### DISCUSSION TITLE
integrationTests of multiple paths

### DISCUSSION THREAD

**Discussion by jopucpla** [apideck-libraries/portman#440] [Q&A]
Hello everyone!

I'm having following issue when trying to combine integrationTests from different parent folders into one unique Integration Test Folder

Here's the scenario:

I have two operations belonging to different folders in the main postman collection (they have different paths and different tags):

- **OperationId:** authenticateUser | **path:** /auth/login
- **OperationId:** logAsUser | **path:** /admin/log-as/connect

When I try to include both in the same integraton test (Case1),

![image](https://user-images.githubusercontent.com/60114539/209327752-4d12c264-223c-4334-a957-e2f70f381ffd.png)

two folders are generated inside Integration Tests folder

![image](https://user-images.githubusercontent.com/60114539/209327850-d1034d49-7640-4ae4-8634-356e6f30bc6e.png)

Is this behavior intended? Is there a way to include in the same folder, so I can test a scenario with multiple operations from different paths? 

I didn't find any other question regarding this topic, please let me know if this has been discussed previously.

Thank you for this brilliant project :)

**Comment by thim81** [comment]
hi @jopucpla 

How are these requests grouped in Postman, for the non-integration requests?

**Reply by jopucpla** [reply]
hi @thim81 

They're generated in different folders (since I'm organizing per tags and they have different tags)

![image](https://user-images.githubusercontent.com/60114539/209348301-7f596d59-615a-435c-8e12-ebe2f575315c.png)
